examples/ufo_toy.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright 2019 the Runebender authors.
+
+//! A quick demonstration of loading and displaying a UFO glyph.
+
+use kurbo::{Affine, Circle, Line, BezPath, Vec2};
+use piet::{RenderContext, FillRule};
+use norad::{glyph::{PointType, Contour, ContourPoint}, Glyph};
+
+use druid_shell::platform::WindowBuilder;
+use druid_shell::win_main;
+
+use druid::widget::Widget;
+use druid::{
+    BoxConstraints, Geometry, Id, LayoutCtx, LayoutResult, PaintCtx,
+    UiMain, UiState,
+};
+
+struct GlyphEditor {
+    glyph: Glyph,
+    path: BezPath,
+    height: f32,
+}
+
+impl GlyphEditor {
+    fn new(glyph: Glyph) -> Self {
+        // assume glyph height is 1000 or 2000 units
```

Comment:
Do you want "unitsPerEm" here?

Reply:
I really don't know what I want; this was just a hack to make sure that glyphs were visible. I'm not familiar with the conventions, but noticed my various glyphs seemed to have different coordinate spaces.

I assume the correct way to deal with this is to go into the fontinfo file and get the cap-height + descender &c, but I don't currently have a way to load that.

Reply:
What do you mean different coordinate spaces? Outline points have absolute coordinates, so you draw them on a canvas and add horizontal lines at the descender, baseline, x, cap and ascender heights and vertical lines at x=0 and x=glyph.width :) The unitsPerEm value then gives you the scale (if you take the same absolute coordinates at 1000 upM and change unitsPerEm to 2048, it looks like you reduced the glyph size).